Lindhart.Analyser.MissingAwaitWarning/Lindhart.Analyser.MissingAwaitWarning/Lindhart.Analyser.MissingAwaitWarning/LindhartAnalyserMissingAwaitWarningAnalyzer.cs

```diff
@@ -74,7 +75,7 @@ private void AnalyseSymbolNode(SyntaxNodeAnalysisContext syntaxNodeAnalysisConte
                         // Checks if a task is not awaited when the task itself is assigned to a variable.
                         case EqualsValueClauseSyntax _:
 
-                            if (EqualsType(methodSymbol.ReturnType, syntaxNodeAnalysisContext.SemanticModel, AwaitableTypes))
+                            if (EqualsType(methodSymbol.ReturnType, syntaxNodeAnalysisContext.SemanticModel, AwaitableTypes) && methodSymbol.IsAsync)
```

Comment:
If the method you are calling is on an interface you don't know if the implementation is `async`. Also on a class you could have a method which is not `async`, which returns a `Task`, which should be awaited in a normal flow.